potoq.go

```diff
@@ -3,14 +3,14 @@ package potoq
 import (
 	"fmt"
 	"github.com/Craftserve/potoq/packets"
-	log "github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus"
```

Comment:
weź te importy podziel na te stdlib, wewnetrzne i zewnetrzne jak w innych modulach